requirements/base.txt

```diff
@@ -2,8 +2,8 @@ dj-static==0.0.6
 gunicorn==19.4.3
 Django==1.8.18
 whitenoise==2.0.6
-psycopg2==2.7.7
-dj-database-url==0.3.0
+psycopg2-binary==2.8.3
```

Comment:
Should be `psycopg2` for prod and (optionally) `psycopg2-binary` for local dev, I think:

> The binary package is a practical choice for development and testing but in production it is advised to use the package built from sources.

https://pypi.org/project/psycopg2/

Reply:
I'll just change to use "psycopg2" all the time.